src/sentry/static/sentry/less/base.less

```diff
@@ -1,27 +1,227 @@
-// stylelint-disable
+*,
```

Comment:
From here on out this file will contain all the styles for our HTML elements. Most of Bootstrap can probably be refactored into this file. Anything with a class should live in a styled component.